shardingsphere-infra/shardingsphere-infra-binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/table/TablesContext.java

```diff
@@ -38,30 +38,30 @@
 @Getter
 @ToString
 public final class TablesContext {
-    
+
```

Comment:
Please do not modify the indent.   
Refer to https://shardingsphere.apache.org/community/cn/contribute/code-conduct/

Reply:
Ok,This is my first pr, I'll notice it, Let me open pr up again

Reply:
Yes，everything is difficult at the beginning, please go on 👍

Reply:
thanks